ILRepack/RepackOptions.cs

```diff
@@ -224,8 +226,13 @@ void Parse()
             LogVerbose = cmd.Modifier("verbose");
             LineIndexation = cmd.Modifier("index");
 
-            // everything that doesn't start with a '/' must be a file to merge (verify when loading the files)
-            InputAssemblies = cmd.OtherAguments;
+			RepackDropAttribute = cmd.Option("repackdrop");
+			if (String.IsNullOrWhiteSpace(RepackDropAttribute)) {
+				RepackDropAttribute = "RepackDropAttribute";
```

Comment:
This will *always* drop the types flagged with the attribute, I think we want to only do this if the option is enabled. (/repackdrop or /repackdrop:attribute)

Reply:
OK, I agree with that.